persist-service/conf.go

```diff
@@ -8,12 +8,15 @@ import (
 )
 
 type Conf struct {
+	Backend BackendType
```

Comment:
Backend 1 is not really user friendly or understandable and depends on the order of the const which is tricky. I'd rather have "Backend": "cassandra" or "file", probably need to add some logic into the newConf()